pkg/csi/cinder/nodeserver.go

```diff
@@ -61,18 +59,21 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 
 	// Volume Mount
 	if notMnt {
-		// Get Options
-		var options []string
+		// Perform a bind mount
+		options := []string{"bind"}
+		fsType := "ext4"
```

Comment:
ext4 might need a configuration? or a TODO here?

Reply:
@jichenjc here, im adding default as ext4 , if none is specified. I'm retrieving configured fstype in L70 -74 below.

Reply:
weird... I think I removed this comment after further review... sorry for bothering.